www/src/components/docs/exampleOptionForm.astro

```diff
@@ -0,0 +1,91 @@
+---
+const options = {
+  nextAuth: "NextAuth.js",
+  prisma: "Prisma",
+  tailwind: "Tailwind CSS",
+  trpc: "tRPC",
+  drizzle: "drizzle",
+};
+---
+
+<script>
+  import sdk from "@stackblitz/sdk";
+
+  function directToStackBlitz(checkedInputs: any) {
+    try {
+      sdk.openProject(
+        {
+          title: "T3 Examples",
+          description: "T3 example apps.",
+          template: "node",
+          files: {
+            "package.json": `{
+                    "name": "T3 Examples",
+                    "version": "1.0.0",
+                    "description": "My Next.js app",
+                    "scripts":{
+                        "dev": " echo 'y' | npx create-t3-app example-app --CI ${checkedInputs} && cd ./my-app  && rm ../package.json " }
```

Comment:
it's currently scaffolding in `./example-app` and then tries to cd to `./my-app`. i think out of the two i prefer `./example-app`

Reply:
Yeah, 😅 That was a typo on my part. I'll get that fixed